tests/installation.bats

```diff
@@ -14,9 +14,8 @@ function setup {
     DHOST=127.0.0.1
     # Extract the IP address we need from DOCKER_HOST, which is formatted like tcp://192.168.99.100:2376
     if [ ! -z "${DOCKER_HOST:-}" ]; then DHOST="$(echo ${DOCKER_HOST} | perl -p -e 's/(tcp:\/\/|:[0-9]+$)//g')"; fi
-
-    cd ${SPRINTDIR} && export SPRINT_NAME=$(./start_sprint.sh)
-    cd ${SPRINTDIR}/${SPRINT_NAME} && echo y | ./start_clean.sh
+    cd ${SPRINTDIR} && ./start_sprint.sh
```

Comment:
It's nice that this was the only test change, and I think that means the tests are working out well.

Reply:
> docker is running, continuing.
> chmod: cannot access ‘/home/circleci/tmp/drupal_sprint_package/sprint’: No such file or directory

I noticed that the chmod error came back in the test run (or my memory is bad and it hadn't been fixed yet):  https://github.com/drud/quicksprint/blob/9b919d961b4896ded59d4b2ee048d735449d0c7f/tests/test_drupal_quicksprint.sh#L31

I'm not sure why that would come up since there is a `mkdir -p` immediately preceding it.